### DISCUSSION TITLE
"Explore repositores" suggestion engine is freaking out

### DISCUSSION THREAD

**Discussion by dragoncoder047** [community/community#68172] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/101021094/4ddfd9a7-d9b4-437c-ae4a-5d6847145454)![image](https://github.com/community/community/assets/101021094/3767c365-9ff3-4adf-844c-b37ce6a99915)![image](https://github.com/community/community/assets/101021094/16c2f1b2-3f86-4929-9565-413b75fb2833)![image](https://github.com/community/community/assets/101021094/b88aa4a0-95f5-40a1-90bd-dcccd0fd9d82)![image](https://github.com/community/community/assets/101021094/80b209de-86d5-4778-8303-27d0965c8d04)

Lots and lots of duplicate entries... even in the same set. It hasn't started giving me duplicates much at all until only recently.

These three screenshots were all taken on consecutive reloads of the home page. I would think I would get 15 different repositories in that case, but no, I got the same 3 repositories, just shuffled around, and sometimes the same repository even was given multiple entries of the same set. I literally reloaded the page like 20 more times, and got nothing but random combinations of valhalla/valhalla, bitwarden/clients, and labwc/labwc.

Two weeks ago I was getting a more diverse selection of repositories. What is going on right now?

**Comment by Zaczero** [comment]
At this point, I wouldn't be surprised if this is yet another "improvement" from GitHub.

**Comment by dragoncoder047** [comment]
Looks like they got it fixed now.
